crates/sui-framework/sources/governance/staking_pool.move

```diff
@@ -0,0 +1,253 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module sui::staking_pool {
+    use sui::balance::{Self, Balance, Supply};
+    use sui::sui::SUI;
+    use std::option::{Self, Option};
+    use sui::tx_context::{Self, TxContext};
+    use sui::transfer;
+    use sui::epoch_time_lock::{EpochTimeLock};
+    use sui::object::{Self, UID};
+    use sui::locked_coin;
+    use sui::coin;
+
+    friend sui::validator;
+
+    const DELEGATION_ALREADY_ACTIVATED: u64 = 0; 
```

Comment:
We have a convention to name errors in ECamelCase, so here and below it would be: 
```move
const EDelegationAlreadyActivated: u64 = 0;
```